Documentation/DataProcessing/_RecordTransformationProcessor/_WithDatabaseFluidTemplate.typoscript

```diff
@@ -0,0 +1,10 @@
+# tt_content.text = FLUIDTEMPLATE
+tt_content.text {
+    templateName = Text
+    dataProcessing {
+        10 = record-transformation
+        10 {
+            as = data
+        }
+    }
```

Comment:
2 letter indent for typoscript:
https://github.com/TYPO3/typo3/blob/main/.editorconfig#L51

```suggestion
  templateName = Text
  dataProcessing {
    10 = record-transformation
    10 {
      as = data
    }
  }
```